examples/app.py

```diff
@@ -87,17 +88,12 @@
    $ flask -a app.py run
```

Comment:
Since you have exported the application here: https://github.com/inveniosoftware/invenio-previewer/pull/40/files#diff-86d2f29b7151f1dfb6b158f5f495ba35R33, you won't need to specify the application again. Actually, I think but I am not sure, ``flask`` won't recognize the ``-a`` parameter ;)